# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/gc/GenerateStatsPhases.py
# Commit: b1117df3b91d
# Full Hash: b1117df3b91d10d68d10b399637cf9ac295057e6
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2019-11-01 16:13:11
# Regressor Bug: 1587096
# File Overlap Count: 1
# Description:
#   Bug 1587096 - Part 3: Implmement FinalizationGroup r=sfink
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49946
# ==============================================================================

diff -r cefc0474e13c -r b1117df3b91d js/src/gc/GenerateStatsPhases.py
--- a/js/src/gc/GenerateStatsPhases.py	Fri Nov 01 10:37:40 2019 +0000
+++ b/js/src/gc/GenerateStatsPhases.py	Fri Nov 01 10:37:47 2019 +0000
@@ -138,6 +138,7 @@
             PhaseKind("SWEEP_LAZYSCRIPTS", "Sweep LazyScripts", 71),
             PhaseKind("SWEEP_WEAKMAPS", "Sweep WeakMaps", 63),
             PhaseKind("SWEEP_UNIQUEIDS", "Sweep Unique IDs", 64),
+            PhaseKind("SWEEP_FINALIZATION_GROUPS", "Sweep FinalizationGroups", 74),
             PhaseKind("SWEEP_JIT_DATA", "Sweep JIT Data", 65),
             PhaseKind("SWEEP_WEAK_CACHES", "Sweep Weak Caches", 66),
             PhaseKind("SWEEP_MISC", "Sweep Miscellaneous", 29),